Source/WTF/wtf/text/TextStream.h

```diff
@@ -240,26 +240,50 @@ TextStream& operator<<(TextStream& ts, const Markable<T, Traits>& item)
     return ts << "unset";
 }
 
-template<typename ItemType, size_t inlineCapacity>
-TextStream& operator<<(TextStream& ts, const Vector<ItemType, inlineCapacity>& vector)
+template<typename SizedContainer>
```

Comment:
Another way to do this is to define `SizedContainer` as a concept, and then use `template<SizedContainer SizedContainerType>`.

Reply:
I played with this, but couldn't get a concept that worked in all cases (kept getting compile errors about DataSegmentVectorEntry use with TextStream). Will give it another try once this lands.